src/main/java/tr/com/srdc/cda2fhir/transform/CCDTransformerImpl.java

```diff
@@ -208,16 +204,18 @@ public Bundle transformDocument(ClinicalDocument cda) {
 
         // transform the sections
         for(Section cdaSec: ccd.getSections()) {
+        	System.out.println(cdaSec.getClass());
             SectionComponent fhirSec = resTransformer.tSection2Section(cdaSec);
```

Comment:
Remove System.out.println's. There is a logger if you need.

Reply:
Got it